rsyslog/overlays/prod/secrets/kafka-cert.enc.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: ENC[AES256_GCM,data:EFo=,iv:gh0/qFGqpgoU0AiNCuinhIafJ6JjZK+yxRSAuiHZ8m0=,tag:pKSxf3ui/bVlf8wrjOoq5Q==,type:str]
+kind: ENC[AES256_GCM,data:A0TfR7IJ,iv:hs1ra3yBeEJ5bGmKyVFWwF+QjKIRzhJjkENmHXpR+D4=,tag:tuairdDHTH8YYsJqtVuIjg==,type:str]
+metadata:
+    name: ENC[AES256_GCM,data:0HBDQKrW20dVsw==,iv:C+7yp8fCPRsl0p6C5NYyVPyDjhrrHyYG8afUla6qx2o=,tag:FV0YmcU0+6M5sFjJR7I79A==,type:str]
```

Comment:
Does the name really need to be encrypted?

Reply:
This was just the output of `sops -e overlays/prod/secrets/kafka-cert.yaml > overlays/prod/secrets/kafka-cert.enc.yaml`.

Is there a way to not have those fields encrypted?